internal/v1/api.go

```diff
@@ -849,6 +854,12 @@ type Timezone struct {
 	Timezone *string `json:"timezone,omitempty"`
 }
 
+// UploadPackageRequest defines model for UploadPackageRequest.
+type UploadPackageRequest struct {
+	Packages            *[]string `json:"packages,omitempty"`
+	RecommendedPackages *int32    `json:"recommendedPackages,omitempty"`
+}
+
```

Comment:
The endpoint must change the component name, `UploadPackageRequest` is some sort of copy-paste leftover.